ledger/src/lib.rs

```diff
@@ -67,6 +72,84 @@ impl Ledger {
         }
     }
 
+    #[instrument(name = "ledger.adjust_exchange_position", skip(self, tx))]
+    pub async fn adjust_exchange_position(
+        &self,
+        tx: Transaction<'_, Postgres>,
+        usd_cents_amount: rust_decimal::Decimal,
+        exchange_position_id: uuid::Uuid,
+        exchange_id: String,
+        instrument_id: String,
+    ) -> Result<(), LedgerError> {
+        let current_balance = self
+            .balances()
+            .exchange_position_account_balance(exchange_position_id)
+            .await?
+            .map(|b| b.settled())
+            .unwrap_or(rust_decimal::Decimal::ZERO);
```

Comment:
should we return an error here? if the option is None or does the unwrap_or pattern is fine?

Reply:
This is fine